app/views/partials/droits-list.html

```diff
@@ -0,0 +1,27 @@
+<accordion close-others="false">
+  <div class="row">
+    <div ng-repeat="droit in droits">
+      <div ng-if="$index % 3 === 0" class="clearfix"></div>
+      <div class="panel-non-eligible">
+        <accordion-group is-open="droit.open">
+          <accordion-heading>
+            <div class="row">
+              <div class="col-xs-10">{{ droit.label }}</div>
+              <div class="col-xs-2">
+                <i class="fa" ng-class="{'fa-chevron-down': droit.open, 'fa-chevron-right': !droit.open}"></i>
+              </div>
+            </div>
+          </accordion-heading>
+          <p>{{ droit.description }}</p>
+          <a
+            href="{{ droit.link }}"
+            class="btn btn-sm btn-default"
+            target="_blank">
+            <i class="fa fa-external-link"></i>
+            Plus d'informations
+          </a>
+        </accordion-group>
+      </div>
+    </div>
+  </div>
+</accordion>
```

Comment:
Missing newline at EoF. Use an editor plugin to support the [EditorConfig](http://editorconfig.org) setup we have.